src/core/ext/filters/client_channel/lb_policy.h

```diff
@@ -272,6 +272,7 @@ class LoadBalancingPolicy : public InternallyRefCounted<LoadBalancingPolicy> {
 
   grpc_pollset_set* interested_parties() const { return interested_parties_; }
 
+  // Note: This must be invoked while holding the combiner.
```

Comment:
I was going to suggest updating tests that directly instantiate `Resolver` instances to make sure to bounce into the combiner before orphaning, but then realized that 
`resolver_component_test`, `cancel_ares_query_test`, `dns_resolver_connectivity_test`, `fake_resolver_test`, and `sockaddr_resolver_test` are all already calling `StartLocked` without the combiner (but I suspect getting away with it in those tests since everything is happening on the main thread anyways).

If not addressed in this PR (looks like a pre-existing issue), maybe a TODO?

Reply:
Yeah, those tests are already a little messy about this, but I think that's actually okay.  The structure of the tests guarantees the same synchronization guarantee that grabbing the combiner explicitly would, so there's no need for the extra machinery.